flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/functions/sql/FlinkSqlOperatorTable.java

```diff
@@ -84,39 +84,94 @@ public static synchronized FlinkSqlOperatorTable instance(boolean isBatchMode) {
 
             // register functions based on batch or streaming mode
             final FlinkSqlOperatorTable finalInstance = instance;
-            dynamicFunctions(isBatchMode).forEach(f -> finalInstance.register(f));
```

Comment:
can we update the docs to advertise we now support the bracketed time functions?